Scripts/MNISTdataset.swift

```diff
@@ -0,0 +1,97 @@
+/*
+Usage:
+
+downloadAndExtract()
+let (trainX, trainY, testX, testY) = readMNIST(trainImages: constants["trainImages"]!, 
+                                                   trainLabels: constants["trainLabels"]!, 
+                                                   testImages: constants["testImages"]!, 
+                                                   testLabels: constants["testLabels"]!)
+*/
+
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
```

Comment:
I think that `fileprivate` would be a good access level for all of these. If this code ever gets put in a module, these are probably not things that we want to expose to other files in the module.

Though I echo @saeta's point that it would be nice to do all of these using swift packages. You could probably add some dependencies to Package.swift.